Dockerfiles/entrypoint.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# operator-bundle-image validate
+
+ansible-playbook -vvvv --connection local -i localhost, operator-test-playbooks/validate-operator-bundle.yml -e"work_dir=/project/output" -e"operator_work_dir=/project/operator-bundle/"
```

Comment:
Will consumers be interested in list of generated artifacts ?

Reply:
I think so since logs are the only source of errors that can be preserved.